processor/elasticapmprocessor/testdata/ecs/elastic_log/output.yaml

```diff
@@ -0,0 +1,325 @@
+        - key: process.runtime.name
+          value:
+            stringValue: java
+        - key: process.runtime.version
+          value:
+            stringValue: "17"
+        - key: os.type
+          value:
+            stringValue: linux
+        - key: os.description
+          value:
+            stringValue: Linux 5.10
+        - key: os.name
+          value:
+            stringValue: Ubuntu
+        - key: os.version
+          value:
+            stringValue: "22.04"
+        - key: device.id
+          value:
+            stringValue: device-123
+        - key: device.model.identifier
+          value:
+            stringValue: MacBookPro14,1
+        - key: device.model.name
+          value:
+            stringValue: MacBook Pro
+        - key: device.manufacturer
+          value:
+            stringValue: Apple
+        - key: user.id
+          value:
+            stringValue: user-1234
+        - key: user.email
+          value:
+            stringValue: user@example.com
+        - key: user.name
+          value:
+            stringValue: john.doe
+        - key: user.domain
+          value:
+            stringValue: example.com
+        - key: user_agent.original
+          value:
+            stringValue: Mozilla/5.0
+        - key: network.connection.type
+          value:
+            stringValue: wifi
+        - key: network.connection.subtype
+          value:
+            stringValue: wlan
+        - key: network.carrier.name
+          value:
+            stringValue: carrier-name
+        - key: network.carrier.mcc
+          value:
+            intValue: "310"
+        - key: network.carrier.mnc
+          value:
+            intValue: "260"
+        - key: network.carrier.icc
+          value:
+            stringValue: US
+        - key: client.address
+          value:
+            stringValue: 192.168.1.100
+        - key: client.port
+          value:
+            intValue: "8080"
+        - key: source.address
+          value:
+            stringValue: 10.0.0.1
+        - key: source.port
+          value:
+            intValue: "9000"
+        - key: source.nat.ip
+          value:
+            stringValue: 203.0.113.1
+        - key: destination.ip
+          value:
+            stringValue: 172.16.0.1
+        - key: faas.instance
+          value:
+            stringValue: faas-instance-1
+        - key: faas.name
+          value:
+            stringValue: my-function
+        - key: faas.version
+          value:
+            stringValue: 1.0.0
+        - key: faas.trigger
+          value:
+            stringValue: http
+        - key: faas.coldstart
+          value:
+            boolValue: false
+        - key: labels.faas_trigger_request_id
+          value:
+            stringValue: req-123
+        - key: faas.execution
+          value:
+            stringValue: exec-456
+        - key: data_stream.dataset
+          value:
+            stringValue: apm.app.test_service
```

Comment:
[for reviewer] note: the service specific data_stream.dataset. This is expected and consolidates the test cases under the delete `elastic_apm`